tiles/src/main/java/com/protomaps/basemap/layers/Roads.java

```diff
@@ -22,7 +22,25 @@ public String name() {
   public void processFeature(SourceFeature sourceFeature, FeatureCollector features) {
     if (sourceFeature.canBeLine() && sourceFeature.hasTag("highway") &&
       !(sourceFeature.hasTag("highway", "proposed", "abandoned", "razed", "demolished", "removed", "construction"))) {
+
       String highway = sourceFeature.getString("highway");
+      String shield_text = sourceFeature.getString("ref");
+      String network_val = sourceFeature.getString("network");
+      shield_text = (shield_text == null ? null : shield_text.split(";")[0]);
```

Comment:
We just want the first ref, not list of refs as is often done in OSM on ways.

Better way is to walk the relations and rank order them, but later...